conanfile.py

```diff
@@ -20,13 +20,16 @@ class DiceTemplateLibrary(ConanFile):
     no_copy_source = True
     options = {
         "with_test_deps": [True, False],
+        "with_svector": [True, False],
     }
     default_options = {
         "with_test_deps": False,
+        "with_svector": False,
```

Comment:
Unrelated: should I also add boost as an optional dep? (For offset pointer stuff in polymorphic_allocator.hpp)

Reply:
How does it work if someone somewhere manually specifies `with_svector=False`? Is it then false for every dependency, or does `=True` overwrite it again?

Asking because this issue with `boost:header_only=True`. I don't think its the exact same issue (because `boost:header_only` is a non-additive feature) but just want to make sure

Reply:
It can be overwritten again.

And boost should also be optional.